modules/gltf_blend/editor_scene_importer_blend.cpp

```diff
@@ -0,0 +1,228 @@
+/*************************************************************************/
+/* Copyright (c) 2007-2021 Juan Linietsky, Ariel Manzur.                 */
+/* Copyright (c) 2014-2021 Godot Engine contributors (cf. AUTHORS.md).   */
+/*                                                                       */
+/* Permission is hereby granted, free of charge, to any person obtaining */
+/* a copy of this software and associated documentation files (the       */
+/* "Software"), to deal in the Software without restriction, including   */
+/* without limitation the rights to use, copy, modify, merge, publish,   */
+/* distribute, sublicense, and/or sell copies of the Software, and to    */
+/* permit persons to whom the Software is furnished to do so, subject to */
+/* the following conditions:                                             */
+/*                                                                       */
+/* The above copyright notice and this permission notice shall be        */
+/* included in all copies or substantial portions of the Software.       */
+/*                                                                       */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,       */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF    */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.*/
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY  */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,  */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
+/*************************************************************************/
+
+#include "core/error/error_list.h"
+#include "core/error/error_macros.h"
+#include "core/io/json.h"
+#include "core/variant/dictionary.h"
+#if TOOLS_ENABLED
+#include "editor_scene_importer_blend.h"
+
+#include "core/core_bind.h"
+#include "core/object/ref_counted.h"
+#include "modules/gltf/editor_scene_importer_gltf.h"
+#include "scene/3d/node_3d.h"
+#include "scene/animation/animation_player.h"
+#include "scene/resources/animation.h"
+
+uint32_t EditorSceneFormatImporterBlend::get_import_flags() const {
+	return ImportFlags::IMPORT_SCENE | ImportFlags::IMPORT_ANIMATION;
+}
+
+void EditorSceneFormatImporterBlend::get_extensions(
+		List<String> *r_extensions) const {
+	r_extensions->push_back("blend");
+}
+
+Node *EditorSceneFormatImporterBlend::import_scene(
+		const String &p_path, uint32_t p_flags, const Dictionary &p_options,
+		int p_bake_fps, List<String> *r_missing_deps, Error *r_err) {
+	String addon_path = EditorSettings::get_singleton()->get_setting(
+			"filesystem/blend/blender_path");
+	String json_parameters =
+			"{\"source\": \"GODOT_SOURCE\", \"sink\": \"GODOT_SINK\"}";
+	String texture_dir_string = p_options["blend/materials/texture_directory"];
```

Comment:
I think this is a mistake, if you use the keep textures option, whathever you export will still reference the proper file on disk, so you should not bother about copying textures or anything like this.

Reply:
Resolved in the next push.